web/themes/joinup/prototype/sass/base/_ordered_list.sass

```diff
@@ -6,7 +6,7 @@
 
 article
   ol
-    padding-left: 0
+    padding-left: 15px
```

Comment:
Unfortunately, this is not a sufficient fix to the reported issue:

![bullets-still-wrong](https://user-images.githubusercontent.com/1923476/28817279-fdd3930c-76a7-11e7-84da-36d0e37114c0.png)